msal/tests/dev apps/DesktopTestApp/MainForm.cs

```diff
@@ -498,5 +503,95 @@ private IEnumerable<string> SplitScopeString(string scopes)
 
             return scopes.Split(new[] { " " }, StringSplitOptions.None);
         }
+
+        private async void b2cLogin_Click(object sender, EventArgs e)
+        {
+            using (new UIProgressScope(this))
+            {
+                ClearResultPageInfo();
+                string[] scopes = { "https://msidlabb2c.onmicrosoft.com/msidlabb2capi/read" };
+                string b2cClientId = "e3b9ad76-9763-4827-b088-80c7a7888f79";
```

Comment:
I thought that clientIDs are read from the LabResponse?

Reply:
i fixed this. had to update to 4.6.2 to be compatible w/the lab. 

---
In reply to: [235680206](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1424#discussion_r235680206) [](ancestors = 235680206)